tests/performance/single_fixed_string_groupby.xml

```diff
@@ -10,8 +10,8 @@
         INSERT INTO perf_lc_fixed_str_groupby SELECT ('number key ' || toString(number % 400)) AS a, ('number key ' || toString(number % 20)) AS b FROM numbers(30000000)
     </fill_query>
 
-    <query>SELECT count() FROM perf_lc_fixed_str_groupby GROUP BY a</query>
-    <query>SELECT count() FROM perf_lc_fixed_str_groupby GROUP BY b</query>
+    <query short="1">SELECT count() FROM perf_lc_fixed_str_groupby GROUP BY a</query>
```

Comment:
Why "short"? It's not logically instantaneous, just increase the data set size. "short" is for queries that cannot be made to run longer.

Reply:
But perf test still works good for these queries...

Reply:
It will work even better for longer queries -- we'll get better signal-to-noise ratio and more precise metrics. So it's worth to make a query longer if it's possible. E.g. this test is marked with `max_ignored_relative_change="0.2"`, but probably we could get 5% precision on longer queries, judging by the other similar `group by` tests.